partiql-planner/build.gradle.kts

```diff
@@ -77,3 +77,54 @@ publish {
     name = "PartiQL Planner"
     description = "PartiQL's Experimental Planner."
 }
+
+// Generate IR
```

Comment:
nit: could make more clear this is for the internal IR

```suggestion
// Generate internal IR
```

Reply:
ack